hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestObjectPut.java

```diff
@@ -248,15 +248,15 @@ public void testCopyObject() throws IOException, OS3Exception {
 
     keyContent = IOUtils.toString(ozoneInputStream, UTF_8);
 
-    Assert.assertEquals(200, response.getStatus());
-    Assert.assertEquals(CONTENT, keyContent);
+    assertEquals(200, response.getStatus());
+    assertEquals(CONTENT, keyContent);
 
     // source and dest same
     try {
       objectEndpoint.put(bucketName, keyName, CONTENT.length(), 1, null, body);
       fail("test copy object failed");
     } catch (OS3Exception ex) {
-      Assert.assertTrue(ex.getErrorMessage().contains("This copy request is " +
+      assertTrue(ex.getErrorMessage().contains("This copy request is " +
```

Comment:
nit: please use assertThrow() here and other tests in this class.

Reply:
Done.